ls_test.go

```diff
@@ -278,12 +280,12 @@ stack {
 							Severity: lsp.DiagnosticSeverityError,
 							Range: lsp.Range{
 								Start: lsp.Position{
-									Line:      4,
-									Character: 2,
+									Line:      6,
```

Comment:
the order of events changed because the "merging/import PR" handles blocks first, then attributes, so the error message of unrecognized comes first.. ideally we should not depend on the order..

Reply:
@katcipis see that this diagnostic is now at line 297.